packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -55,15 +54,6 @@ const SidebarLink: FC<SidebarLinkProps> = (props) => {
   }
 
   if (props.superAdmin && !superAdmin) return null;
-  if (props.permissions) {
-    let allowed = false;
-    for (let i = 0; i < props.permissions.length; i++) {
-      if (permissions.check(props.permissions[i] as GlobalPermission)) {
-        allowed = true;
-      }
-    }
-    if (!allowed) return null;
-  }
```

Comment:
Now, if the link has sublinks, and the user doesn't have permission for any sublinks, we don't show the parent link at all.

This removes the need for us to check permission at both the parent and sublink level.